pkg/trace/api/api.go

```diff
@@ -724,6 +725,9 @@ func tracesFromSpans(spans []pb.Span) pb.Traces {
 // getContainerTag returns container and orchestrator tags belonging to containerID. If containerID
 // is empty or no tags are found, an empty string is returned.
 func getContainerTags(containerID string) string {
+	if containerID == "" {
+		return ""
```

Comment:
these container tags should be available as resource attributes in opentelemetry, i will try to investigate what attributes to use here, but I don't think this is blocking.

Reply:
The container tags are obtained from the orchestrator API (k8s, ECS, etc). I wouldn't count on OTLP clients to send it because I doubt apps want to query it and add latency on the client side. I'd advise we keep these because they are very useful and key to the Datadog product.